src/tracks/ui/TextEditHelper.cpp

```diff
@@ -0,0 +1,666 @@
+/*!********************************************************************
+*
+ Audacity: A Digital Audio Editor
+
+ TextEditHelper.h
+
+ Vitaly Sverchinsky
+
+ The major part of the logic is extracted from LabelTrackView.cpp and
+ LabelTextHandle.cpp
```

Comment:
This logic was DUPLICATED from LabelTrackView and LabelTextHandle.

But why duplicate, when we might really reuse?

Can it be given a better name like "TextEditingCell" and could it first be split out and used in LabelTrackView?

Reply:
It's not a cell because it does not define TrackPanel separation as other cells do.
There is another PR which addresses an issue with code duplication